ui/ui-components/components/property/profilePreview.tsx

```diff
@@ -177,8 +199,9 @@ export default function ProfilePreview(props) {
           {Object.keys(otherProperties).map((k) => (
             <ListGroup.Item key={`profile-preview-row-${k}`} variant="light">
               <strong>{k}</strong>:{" "}
-              {otherProperties[k]?.type === "date"
-                ? new Date(otherProperties[k]?.values[0]).toLocaleString()
+              {otherProperties[k]?.type === "date" &&
+              typeof otherProperties[k]?.values[0] === "number"
```

Comment:
it's a little funny to be falling through this `if` when it's a date but somehow not a number. maybe "if date, then generatePreviewDate" and handle null and otherwise in `generatePreviewDate` 
Not a huge deal per se. Just an idea.